build-system/common/logging.js

```diff
@@ -29,6 +28,16 @@ const dotWrappingWidth = 150;
  */
 let loggingPrefix = '';
 
+/**
+ * Logs messages with a timestamp. The timezone suffix is dropped.
+ * @param  {...string} messages
+ */
+function log(...messages) {
```

Comment:
Nit suggestion, take it or leave it - rename function to something more unique like `logWithTimestamp` to avoid naming conflicts in other modules.

Reply:
This one is intentional. There are a few hundred call-sites in `build-system/` that use `log()` today, and I didn't want them to have to change their timestamp-by-default calling convention. There already exists a non-timestamp version called `logWithoutTimestamp()`, which is explicitly used in a few dozen places.